README.md

```diff
@@ -205,6 +217,8 @@ export default function App() {
 | selectedIconProps      | Object   |          | Custom style props based on chakra-ui for the green tick icon in dropdown list, Example `{{ bg: 'gray.100', pt: '4'}}                                            |
 | icon      | Object   |   CheckCircleIcon       | @chakra-ui/icons Icon to be displayed instead of CheckCircleIcon                                            |
 | hideToggleButton      | boolean   |          | Hide the toggle button                                         |
+| disableCreateItem      | Object   |          | Disable the "create new"  list Item. Default is `false`                                             |
```

Comment:
I believe `disableCreateItem` is a boolean value

Reply:
Oh, sorry for the typo!
I just pushed the fix commit.